oci/index.rst

```diff
@@ -1,82 +1,64 @@
-Ubuntu on OCI Registries
-========================
+Ubuntu on OCI container registries
```

Comment:
Should the initials be capitalized?

Reply:
I don't think there's a strict rule on that, as long as it is consistent (which it seems to be) with the remaining titles.

Reply:
The Canonical style guide advocates for using sentence case in headers (meaning this instance is correct). However, when I look at the rendered docs preview, I can see that the entries in the top left both use capitalisation (and shouldn't). I think these are both set in the custom_conf.py file?

![image](https://github.com/canonical/ubuntu-cloud-docs/assets/116022293/d135b082-c09b-4baf-a4d4-a2aaf9ca5e2b)

Reply:
I think the documentation title is defined in [oci/conf.py](https://github.com/canonical/ubuntu-cloud-docs/pull/213/files#diff-1cf1472604d6c6cd298ad0906a73995834f2b04fc0007da94d91faf5736ba572), which seems to have the initials capitalized.

Reply:
Good catch. Fixed in https://github.com/canonical/ubuntu-cloud-docs/pull/213/commits/2cbb3a2c3995bfbbc64ab985507cd2c2b2498c3c